pkg/resources/spark/configmap.go

```diff
@@ -36,15 +36,16 @@ func NewFrameworkConfigMap(sc *dcv1alpha1.SparkCluster) *corev1.ConfigMap {
 // NewKeyTabConfigMap generates a configmap which represents the Kerberos KeyTab configuration out of provided config
 func NewKeyTabConfigMap(sc *dcv1alpha1.SparkCluster) *corev1.ConfigMap {
 	binaryData := map[string][]byte{}
-	if sc.Spec.Master.KeyTabConfig != nil {
-		binaryData[string(ComponentMaster)] = sc.Spec.Master.KeyTabConfig.KeyTab
-	}
-	if sc.Spec.Worker.KeyTabConfig != nil {
-		binaryData[string(ComponentWorker)] = sc.Spec.Worker.KeyTabConfig.KeyTab
+
+	if sc.Spec.KerberosKeytab != nil {
+		binaryData[string(ComponentMaster)] = sc.Spec.KerberosKeytab.Contents
+		binaryData[string(ComponentWorker)] = sc.Spec.KerberosKeytab.Contents
```

Comment:
I just wonder if we have to maintain two separate keys for master/worker with the same keytab infos? Should we condense this into a single field (e.g. `"keytab": []byte("dem contents")`  and remove these extra bits?

See https://github.com/dominodatalab/distributed-compute-operator/blob/3aeef3d7a76c3b4f3e21cd36b7ca38ad62c1dc86/pkg/cluster/dask/configmap.go#L36